ia/src/ml_agent.py

```
@@ -0,0 +1,155 @@
+import protocole
+import game
+import math
+
+def get_position(index, total_cells) -> tuple[int, int]:
```

Ruff (0.11.9)

Missing type annotation for function argument `index`

(ANN001)